### PR TITLE
move isRTL initialization before its first use inside the if

### PR DESCRIPTION
**Pull request by bauna**

### CODE REVIEW

File: plugin/src/main/java/com/arcbees/chosen/client/ChosenImpl.java [hunk 1]
```diff
@@ -1504,7 +1505,7 @@ private void setup() {
             isHidden = fWidth > 0;
         }
 
-        isRTL = LocaleInfo.getCurrentLocale().isRTL() || $selectElement.hasClass("chzn-rtl");
+
```
Comment (jDramaix) [inline plugin/src/main/java/com/arcbees/chosen/client/ChosenImpl.java]: rm 1508 & 1509

**Comment by jDramaix** [comment]
Good catch. Please fix checkstyle by removing extra new lines.

**Comment by jDramaix** [comment]
And thanks for this contribution

**Comment by bauna** [comment]
it's done. 
Thank you guys!

**Pull request merged**